x-pack/plugins/ml/public/data_frame/components/aggregation_list/list_form.tsx

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+
+import { i18n } from '@kbn/i18n';
+
+import {
+  EuiFieldText,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFormRow,
+  EuiListGroup,
+  EuiListGroupItem,
+} from '@elastic/eui';
+
+import { Dictionary } from '../../../../common/types/common';
+
+interface OptionsDataElement {
+  agg: string;
+  field: string;
+  formRowLabel: string;
+}
+
+interface ListProps {
+  list: string[];
+  optionsData: Dictionary<OptionsDataElement>;
+  deleteHandler?(l: string): void;
+}
+
+export const AggListForm: React.SFC<ListProps> = ({ deleteHandler, list, optionsData }) => (
+  <EuiListGroup flush={true}>
+    {list.map((l: string) => (
```

Comment:
A more descriptive variable name than just `l` would be useful here. For example, it is used in the `aria-label` by which point it is unclear what `l` refers to

Reply:
Changed to `optionsDataId` in 6b4b1f4.